pkgs/development/python-modules/jaxlib/bin.nix

```diff
@@ -68,96 +68,102 @@ let
+        hash = "sha256-vilavCCcmAgX2wSI8h8fvAZE+HMmUiiV4rm2RykQY1c=";
+      };
+      "3.12-aarch64-linux" = getSrcFromPypi {
+        platform = "manylinux2014_aarch64";
+        dist = "cp312";
+        hash = "sha256-+7q6ooc3inijz5y+TeMKH28ZqZEW/rS9aH/yVkFc1EI=";
       };
       "3.12-aarch64-darwin" = getSrcFromPypi {
         platform = "macosx_11_0_arm64";
         dist = "cp312";
-        hash = "sha256-jdi//jhTcC9jzZJNoO4lc0pNGc1ckmvgM9dyun0cF10=";
+        hash = "sha256-VZeQjNEEGMC0Lpr4B/yBEgNnA1M89QGlJVqPv0ARhn4=";
       };
       "3.12-x86_64-darwin" = getSrcFromPypi {
         platform = "macosx_10_14_x86_64";
         dist = "cp312";
-        hash = "sha256-1sCaVFMpciRhrwVuc1FG0sjHTCKsdCaoRetp8ya096A=";
+        hash = "sha256-WXKqhfbXcezIzHIUjB+mQlDKM8vfK/JEB83uilKZ0l0=";
       };
-    };
 
-  # Note that the prebuilt jaxlib binary requires specific version of CUDA to
-  # work. The cuda12 jaxlib binaries only works with CUDA 12.2, and cuda11
-  # jaxlib binaries only works with CUDA 11.8. This is why we need to find a
-  # binary that matches the provided cudaVersion.
-  gpuSrcVersionString = "cuda${cudaVersion}-${pythonVersion}";
+      "3.13-x86_64-linux" = getSrcFromPypi {
+        platform = "manylinux2014_x86_64";
+        dist = "cp313";
+        hash = "sha256-k/HFAtCOUX+EL+exhCi7CGz9B32w6pokGPsh5bTgbT0=";
+      };
+      "3.13-aarch64-linux" = getSrcFromPypi {
+        platform = "manylinux2014_aarch64";
+        dist = "cp313";
+        hash = "sha256-fOk2hRU0jYadbFnZkEw8s8gfIv8+npaerg41Y/5HIIA=";
+      };
+      "3.13-aarch64-darwin" = getSrcFromPypi {
+        platform = "macosx_11_0_arm64";
+        dist = "cp313";
+        hash = "sha256-FuyYDoWYP0GZnE3IQTfexwUH2VjiPX7voQTakwU9E18=";
+      };
+      "3.13-x86_64-darwin" = getSrcFromPypi {
+        platform = "macosx_10_14_x86_64";
+        dist = "cp313";
+        hash = "sha256-At+cDhMj3eAelmwi6xJDKQXS1N6KrHtgPK0ggxAbDms=";
+      };
+    };
 
   # Find new releases at https://storage.googleapis.com/jax-releases
   # When upgrading, you can get these hashes from prefetch.sh. See
   # https://github.com/google/jax/issues/12879 as to why this specific URL is the correct index.
   gpuSrcs = {
-    "cuda12.2-3.9" = fetchurl {
-      url = "https://storage.googleapis.com/jax-releases/cuda12/jaxlib-${version}+cuda12.cudnn89-cp39-cp39-manylinux2014_x86_64.whl";
-      hash = "sha256-d8LIl22gIvmWfoyKfXKElZJXicPQIZxdS4HumhwQGCw=";
-    };
-    "cuda12.2-3.10" = fetchurl {
-      url = "https://storage.googleapis.com/jax-releases/cuda12/jaxlib-${version}+cuda12.cudnn89-cp310-cp310-manylinux2014_x86_64.whl";
-      hash = "sha256-PXtWv+UEcMWF8LhWe6Z1UGkf14PG3dkJ0Iop0LiimnQ=";
-    };
-    "cuda12.2-3.11" = fetchurl {
-      url = "https://storage.googleapis.com/jax-releases/cuda12/jaxlib-${version}+cuda12.cudnn89-cp311-cp311-manylinux2014_x86_64.whl";
-      hash = "sha256-QO2WSOzmJ48VaCha596mELiOfPsAGLpGctmdzcCHE/o=";
+    "x86_64-linux" = fetchurl {
```

Comment:
TODO:
Since CUDA support is now available as a plug-in system, it needs to be added to the dependencies as a separate package.